app/views/searches/_jobs.mobile.haml

```diff
@@ -13,3 +13,5 @@
       = render partial: 'show_more_show_less'
   .content-block-item.content-block-item-footer
     .see-more= link_to_more_jobs search
+
+= search.jobs["SearchResult"]["SearchResultItems"]
```

Comment:
Is this line needed? The view code shouldn't need to change.

Reply:
yikes good catch that is not needed.